lemur/common/celery.py

```diff
@@ -1201,7 +1201,7 @@ def rotate_endpoint(endpoint_id):
     endpoint = endpoint_service.get(endpoint_id)
 
     if not endpoint:
-        raise RuntimeError("endpoint did not exist")
+        logger.info(f"Skipping rotation,due to {endpoint_id} did not exist")
```

Comment:
```suggestion
        logger.info(f"Skipping rotation,due to {endpoint_id} did not exist")
        return
```

Reply:
It make sense thanks for raising my attention.